src/main/java/net/sf/jabref/logic/util/Version.java

```diff
@@ -31,40 +33,38 @@
 public class Version {
 
     public final static String JABREF_DOWNLOAD_URL = "http://www.fosshub.com/JabRef.html";
+    private static final Log LOGGER = LogFactory.getLog(Version.class);
     private final static String JABREF_GITHUB_URL = "https://api.github.com/repos/JabRef/jabref/releases/latest";
 
-    private final String fullVersion;
-    private final int major;
-    private final int minor;
-    private final int patch;
-    private final boolean isDevelopmentVersion;
-
+    private String fullVersion = BuildInfo.UNKNOWN_VERSION;
+    private int major;
+    private int minor;
+    private int patch;
+    private boolean isDevelopmentVersion;
 
     /**
-     * @param version must be in form of X.X (eg 3.3; 3.4dev)
+     * @param version must be in form of X.X (e.g., 3.3; 3.4dev)
      */
     public Version(String version) {
-        if (version == null || "".equals(version) || version.equals(BuildInfo.UNKNOWN_VERSION)) {
-            this.fullVersion = BuildInfo.UNKNOWN_VERSION;
-            this.major = this.minor = this.patch = 0;
-            this.isDevelopmentVersion = false;
+        if ((version == null) || "".equals(version) || version.equals(BuildInfo.UNKNOWN_VERSION)) {
```

Comment:
Can you add `"${version}".equals(version)` as another disjunction?

When you compile the code without gradle (like directly in eclipse or IntelliJ IDEA) the versionnumber is not set (as it is set within the gradle processResources task) and `${version}` will be read in which will currently lead to an error.